packages/synapse-react-client/src/components/EntityHeaderTable/DebouncedInput.tsx

```diff
@@ -16,20 +16,21 @@ export function DebouncedInput({
   options: string[]
   delay?: number
 } & Pick<
-  React.InputHTMLAttributes<TextFieldProps>,
-  'type' | 'min' | 'max' | 'value' | 'placeholder' | 'className' | 'list'
+  TextFieldProps,
+  'type' | 'value' | 'placeholder' | 'className' | 'label'
 >) {
   const [value, setValue] = useState(initialValue)
   useDebouncedEffect(
     () => {
       onChange(value)
     },
-    [value],
+    [onChange, value],
     delay,
   )
 
   return (
     <Autocomplete
+      freeSolo
```

Comment:
This allows typing freely, which is desired. This does not seem to actually change how the user can interact with the text input, but it silences a warning that was logged during tests.

Reply:
😂